examples/extract-table-from-image/index.ts

```diff
@@ -0,0 +1,70 @@
+import Instructor from "@/instructor"
+import OpenAI from "openai"
+import { z } from "zod"
+
+const IMAGE_URL = "https://duquark.files.wordpress.com/2019/12/graph.jpg?w=700&h=499"
+
+const TableResponseSchema = z.object({
+  title: z.string(),
+  caption: z.string(),
+  table: z.any()
+})
+type TableResponse = z.infer<typeof TableResponseSchema>
```

Comment:
not sure you need the extra typing or function wrapper + try/catch in here (extractTableFromImage)

if you just run it all at root, you'll get inference on the response and well-formatted errors on failure

Reply:
sure thing, let me simplify this

Reply:
https://github.com/instructor-ai/instructor-js/pull/118/commits/e7cc8c85cc9922ea34f6af8879e93c996285e13d